Bottom: fe546e7b37e22ec971613e78e81118ff95e37a9b
Top:    ff116f0619286e27e750469abc84aec42eddf294
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:14 +0000

draw.h: Remove unused DrawHoles() prototype


---

diff --git a/src/draw.h b/src/draw.h
index e0688ee..bae0cd8 100644
--- a/src/draw.h
+++ b/src/draw.h
@@ -70,7 +70,6 @@ void DrawLayerGroup (int side, const BoxType *drawn_area);
 void DrawPaste (int side, const BoxType *drawn_area);
 void DrawSilk (HID_DRAW *hid_draw, int side, const BoxType *drawn_area);
 void DrawMask (int side, const BoxType *drawn_area);
-void DrawHoles (bool draw_plated, bool draw_unplated, const BoxType *drawn_area);
 void PrintAssembly (int side, const BoxType *drawn_area);
 
 /* TEMPORARY */
